.circleci/docker/common/install_conda.sh

```diff
@@ -96,26 +94,27 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   conda_install nnpack -c killeent
 
   # Install some other packages, including those needed for Python test reporting
-  # TODO: Why is scipy pinned
+  # Pin SciPy because of failing distribution tests (see #60347)
```

Comment:
Test failures were due to `pip install librosa` also upgrading SciPy, which in turn caused #60347 to re-appear. Workaround for now is to update the SciPy pin.